### PR TITLE
Fixes #17369: Typo in merge policy doc

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/17369

### CODE REVIEW

File: src/reference/modules/usage/pages/advanced_configuration_management.adoc [hunk 1]
```diff
@@ -293,12 +293,12 @@ agent from Directives based on the same Technique.
 
 === Policy generation and Directive merge
 
-In Rudder, Policies are generated from Directives. In the general case, each directive will lead to one policy on the node. 
+In Rudder, policies are generated from directives. In the general case, each directive will lead to one policy on the node. 
 
-There is two exception to that rule:
+There is two exceptions to that rule:
```
Comment (amousset) [inline src/reference/modules/usage/pages/advanced_configuration_management.adoc:298]: There are

File: src/reference/modules/usage/pages/advanced_configuration_management.adoc [hunk 2]
```diff
@@ -293,12 +293,12 @@ agent from Directives based on the same Technique.
...
 
-- for unique (non multi-instance) technique, the directive with the highest priority is selected.
-- for multi-instance technique, if the technique metadata `POLICYGENERATION` tag has value `merged`, then all directives will be merged into *one* policy on the agent. That behavior used to be the default one until Rudder 4.3. See https://issues.rudder.io/issues/10625[Don't merge directive from same technique on generation] for more information.
+- for unique (non multi-instance) technique, the directive with the highest priority is selected and other are discarded (and they will be tagged as such in node detail compliance),
```
Comment (amousset) [inline src/reference/modules/usage/pages/advanced_configuration_management.adoc:300]: others

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**